geniza/templates/nav.html

```diff
@@ -14,7 +14,7 @@
                 {% translate "home" as aria_home %}
                 {# Translators: site name label for 'home' button in main navigation #}
                 {% translate "GENIZA" as site_name %}
-                <a class="home-link" href="#" aria-label="{{ aria_home }}" role="button">{{ site_name }}</a>
+                <a class="home-link" href="/" aria-label="{{ aria_home }}" role="button">{{ site_name }}</a>
```

Comment:
do we need a different link here for root of current multilingual site, or will this work as expected?

Reply:
It's automatically redirecting me to `/en`, but not sure we have a way of changing the language yet on the frontend so I can test with the others. But I couldn't find another way to link to the root page in Wagtail template tags—maybe I missed it!

Reply:
we don't have a way in the UI, I've just been editing the url when I wanted to test. this is probably good enough for now, we can revisit if needed!